wagclientlogger/wagClientLoggerImplementation.go

```diff
@@ -0,0 +1,50 @@
+package wagclientlogger
+
+import (
+	"github.com/Clever/kayvee-go/v7/logger"
+	wcl "github.com/Clever/wag/logging/wagclientlogger"
+)
+
+var kv logger.KayveeLogger
+
+//NewLogger creates a logger for id that produces logs at and below the indicated level.
+//Level indicated the level at and below which logs are created.
+func NewLogger(id string, level wcl.LogLevel) WagClientLogger {
+	newLogger := logger.New(id)
+
+	if level != wcl.FromEnv {
+		newLogger.SetLogLevel(logger.LogLevel(level))
+	}
+
+	kv = newLogger
+
+	return WagClientLogger{id: id, level: level}
+}
+
+type WagClientLogger struct {
+	level wcl.LogLevel
+	id    string
+}
+
+func (w WagClientLogger) Log(level wcl.LogLevel, title string, m map[string]interface{}) {
```

Comment:
The code here looks good, but curious why this needs to be in it's own package? Generally, interfaces are created for exactly this kind of problem. Interfaces don't create explicit dependencies, so if this method was added directly to kayvee then it doesn't matter which version of kayvee that the client consumer imports and passes to wag client. 

One other thing I should flag is that I think you have an unintended dep version  dependency here. You use wcl.LogLevel as a type in this interface. If wcl ever changes major versions then an interface may not be satisfied by a new version of a logger. One possible solution: If the LoLevel is is a simple int type, use that instead then cast internally within the Log method to the actual level.

Reply:
My thinking here was:
Having it as it's own package makes it easier to update it without needing to do a bump of kayvee, and since the interface definition is it's own module in wag it's an easy path to update both. 

It shortens the use case. (wcl.NewLogger() vs logger.wagclientlogger.NewLogger())

Reply:
I originally implemented this as LogLevel being a simple string and then casting it internally to the actual level. 
https://docs.google.com/document/d/184-Gkg_BTrw_t7mhu175eI_O-JvIPZMKCjYywWWZLiU/edit?disco=AAAAcMqT4H0

But it was suggested that I use an enum instead, thus creating the dependency.